recipes/antlr-python-runtime/meta.yaml

```diff
@@ -1,3 +1,6 @@
+{% set name = " " %}
+{% set sha256 = "" %}
```

Comment:
Set them to empty strings by default if a Python version is unspecified (e.g. re-rendering). Otherwise there are complaints about undefined Jinja variables. The content of these values is irrelevant in those cases. Also add a space to `name` to allow `name[0]` to pass later.

Reply:
Great! Certainly a concise way to fix it... is there an even better way to do it that wouldn't force this edge case 😊 ?

Reply:
Probably adding selectors to the `url` and `sha256` fields instead (thus dropping `name` and `sha256` Jinja variables).